### PR TITLE
Adding more aliases

### PR DESCRIPTION
**Pull request by Kdisimone**

### CODE REVIEW

**Comment by PieterGit** [comment]
I think I would prefer not to add a lot more aliases because I think you should learn people to understand what they are doing (even basic Linux skills). This patch is also inconsistent because it uses `vi` for editing one file and `nano` for another. I would limit the aliases to viewing the logs and document the rest of the commands.

I think it can be usefull to maybe create a `oref0-debug` that checks some of the basic errors and reports back the status. Examples: 
- last succesfull pump loop
- last succesfull cgm
- perhaps version information of components 
- ...

**Comment by Kdisimone** [comment]
Respectfully disagree with leaving aliases out.  That doesn't help people learn more.  They will have to learn quite a bit, beyond just these aliases, to setup openaps and an edison.  Perpetually making someone type out a long series of commands for the exact same thing is not making them learn anything (other than that one long series of commands).    I have no preference for `vi` vs `nano`.

**Comment by ELUTE** [comment]
+1 on adding aliases. Makes life so much easier for users.

**Comment by leighkdavis** [comment]
Coming to check to see if there were any more possible aliases and shortcuts avaialable and came across this PR. These are one of my favorite features and have made this process for me much easier to edit and view on the go. Building OpenAPS has been a four month process for me of stops and starts. I have learned so much. Please know that the shortcuts haven't made me learn less about the system....just makes it easier for me to get back in and do what needs to be done with efficiency. Just my two cents.

File: bin/oref0-log-shortcuts.sh [hunk 1]
```diff
@@ -1,13 +1,23 @@
 #!/bin/bash
 
 # add crontab entries
-grep -q networklog ~/.bash_profile 2>/dev/null || echo "alias networklog="'"tail -n 100 -F /var/log/openaps/network.log"' >> ~/.bash_profile
+grep -q network-looplog ~/.bash_profile 2>/dev/null || echo "alias network-looplog="'"tail -n 100 -F /var/log/openaps/network.log"' >> ~/.bash_profile
```
Comment (scottleibrand) [inline bin/oref0-log-shortcuts.sh]: I don't think we want to add "loop" to these, as that's not present in the name of the logfile that's being tailed.

File: bin/oref0-log-shortcuts.sh [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
 grep -q xdrip-looplog ~/.bash_profile || echo "alias xdrip-looplog="'"tail -n 100 -F /var/log/openaps/xdrip-loop.log"' >> ~/.bash_profile
 grep -q cgm-looplog ~/.bash_profile || echo "alias cgm-looplog="'"tail -n 100 -F /var/log/openaps/cgm-loop.log"' >> ~/.bash_profile
 grep -q autosens-looplog ~/.bash_profile || echo "alias autosens-looplog="'"tail -n 100 -F /var/log/openaps/autosens-loop.log"' >> ~/.bash_profile
-grep -q autotunelog ~/.bash_profile || echo "alias autotunelog="'"tail -n 100 -F /var/log/openaps/autotune.log"' >> ~/.bash_profile
+grep -q autotune-looplog ~/.bash_profile || echo "alias autotune-looplog="'"tail -n 100 -F /var/log/openaps/autotune.log"' >> ~/.bash_profile
```
Comment (scottleibrand) [inline bin/oref0-log-shortcuts.sh]: I don't think we want to add "loop" to these, as that's not present in the name of the logfile that's being tailed.

File: bin/oref0-log-shortcuts.sh [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
 grep -q pump-looplog ~/.bash_profile || echo "alias pump-looplog="'"tail -n 100 -F /var/log/openaps/pump-loop.log"' >> ~/.bash_profile
 grep -q urchin-looplog ~/.bash_profile || echo "alias urchin-looplog="'"tail -n 100 -F /var/log/openaps/urchin-loop.log"' >> ~/.bash_profile
+grep -q ns-looplog ~/.bash_profile || echo "alias ns-looplog="'"tail -n 100 -F /var/log/openaps/ns-loop.log"' >> ~/.bash_profile
+grep -q cat-pref ~/.bash_profile || echo "alias cat-pref="'"cd ~/myopenaps && cat preferences.json"' >> ~/.bash_profile
+grep -q edit-pref ~/.bash_profile || echo "alias edit-pref="'"cd ~/myopenaps && vi preferences.json"' >> ~/.bash_profile
```
Comment (scottleibrand) [inline bin/oref0-log-shortcuts.sh]: Let's make these ones use nano for consistency and ease of use.

File: bin/oref0-log-shortcuts.sh [hunk 4]
```diff
@@ -1,13 +1,23 @@
...
+grep -q cat-wifi ~/.bash_profile || echo "alias cat-wifi="'"cat /etc/wpa_supplicant/wpa_supplicant.conf"' >> ~/.bash_profile
+grep -q edit-wifi ~/.bash_profile || echo "alias edit-wifi="'"vi /etc/wpa_supplicant/wpa_supplicant.conf"' >> ~/.bash_profile
```
Comment (scottleibrand) [inline bin/oref0-log-shortcuts.sh]: Let's make these ones use nano for consistency and ease of use.

Comment (Kdisimone) [inline bin/oref0-log-shortcuts.sh]: sounds good

**Comment by Kdisimone** [comment]
I think i've got the nano and vi changes done.  Looks good.  Thank you Scott

**Comment by PieterGit** [comment]
Some implementation details:
- I would honor the basic / standard editor preferences in Unix, e.g. https://wiki.debian.org/TextEditor, e.g. use `$EDITOR preferences.json ` instead `nano preferences.json`
- The `myopenaps` part is now hardcoded in the aliases. The directory the users sets in `oref0-setup.sh` should be used.
- Grepping lines for existence is hard to maintain, better to create a seperate oref0-file and use `source` to import that file
- The previous commit of adding the aliases broke the Nightscout interaction on the CLI which took me quite some time to figure out what had happened. I fixed that on my branch:
https://github.com/PieterGit/oref0/commit/639aa56f930a0b0de9408b092ebd838e23fbf25b

**Pull request merged**